### DISCUSSION TITLE
CategoryController Metodo PUT [RESOLVIDO]

### DISCUSSION THREAD

**Discussion by ernaneSNcarvalho** [balta-io/7194#24] [Perguntas e Respostas] [answered]
Bom dia, 

Estou criando o BD Conforme aula "Conectando ao SqlServer". 
Duvidas: 
1 - La na aula de instalacao do KITEMATIC quando buscou pelo container "mssql-server-linux" no cabecalho havia o nome microsoft, eu busquei bastante por ele e nao encontrei, apesar de haver varios so que de outras distribuicoes (nao sei se e assim que fala). Escolhi um container porem ele nem da opcao de settings com o password (conforme voce copiou e colou no appsettings json). Entao a pergunta e: posso usar o mesmo container que usou nas aulas anteriores? ex: o proprio kitematic ja puxou o que usamos.

2 - As migrations foram geradas sem problemas, mas ao realizar o update, ele diz que "Login failed for user 'SA'.", eu estou usando mesmo login e password das configuracoes iniciais. Como resolver esse erro ?

https://github.com/ernaneSNcarvalho/Shop    CODIGO COMPLETO

**Comment by ernaneSNcarvalho** [comment, marked as answer]
[resolvido]

**Reply by andrebaltieri** [reply]
@ernaneSNcarvalho posta como você resolveu, pode ajudar mais pessoas com o mesmo problema!

**Reply by ernaneSNcarvalho** [reply]
Eu simplesmente conectei ao container que havíamos criado durante a aula.

**Reply by andrebaltieri** [reply]
Perfeito! Obrigado <3
